### ISSUE TITLE
Feature request: Add a single letter shortcut to mute / unmute the simulator

### ISSUE THREAD

**Issue by microbit-robert** [microsoft/pxt-microbit#6617]
**Is your feature request related to a problem? Please describe.**
As part of ongoing work to improve accessibility, it is desirable for some users to be able to mute / unmute the simulator without having to navigate around the app to do so. When adding sound blocks to the `onStart` or `forever` blocks, the simulator automatically restarts and screen reader users can find the source of this sound confusing and the continued output overwhelming.

**Describe the solution you'd like**
We would like the MakeCode team to consider adding a single letter shortcut that can mute / unmute the simulator, or alternatively to stop / restart the simulator. This action should be accompanied by updating an aria-live region to announce the simulator state when it changes. We would be happy to implement this feature and submit a PR if the MakeCode team also thinks this is valuable.

**Describe alternatives you've considered**
It is already possible to mute / unmute or stop and restart the simulator, but this requires a non-trivial amount of navigation for screen reader users as outlined below. Other suggestions are welcome.

- Navigate to the simulator -> mute button
- Navigate back to the workspace -> make code changes
- Navigate back to the simulator to unmute
- Repeat as necessary

**Additional context**
We hope to test with a single letter shortcut to mute / unmute or stop / restart the simulator when conducting user testing with Blockly screen reader improvements. However, it seems unfair to test with such a feature unless the MakeCode team would support its addition if it received positive feedback.

**Comment by microbit-robert** [comment]
Implemented for blocks in https://github.com/microsoft/pxt/pull/11356, but not the monaco editor.

**Comment by microbit-matt-hillsdon** [comment]
We're not currently planning anything beyond the blocks shortcut that went in above. Suggest we close and/or split this issue and get input on what the MakeCode team thinks makes sense for Monaco users here.
